theseus/src/launcher/mod.rs

```diff
@@ -300,26 +273,19 @@ pub async fn launch_minecraft(
             credentials,
             &version.id,
             &version.asset_index.id,
-            root_dir,
+            &root_dir,
             &assets_path,
             &version.type_,
-            settings.game_resolution,
+            *resolution,
         )?)
         .current_dir(root_dir)
         .stdout(Stdio::inherit())
         .stderr(Stdio::inherit());
 
-    let mut child = command.spawn().map_err(|err| LauncherError::ProcessError {
+    command.spawn().map_err(|err| LauncherError::ProcessError {
         inner: err,
         process: "minecraft".to_string(),
```

Comment:
Could we possibly have a more descriptive name of the process? (maybe profile or MC version name)

Reply:
Sure, should be feasible enough.